app/src/main/res/layout/item_media_preview.xml

```diff
@@ -16,14 +16,15 @@
         android:id="@+id/status_sensitive_media_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:alpha="0.7"
+        android:layout_margin="3dp"
+        android:background="@drawable/media_warning_bg"
         android:contentDescription="@null"
         android:padding="@dimen/status_sensitive_media_button_padding"
         android:visibility="gone"
         app:layout_constraintStart_toStartOf="@+id/status_media_preview_container"
         app:layout_constraintTop_toTopOf="@+id/status_media_preview_container"
         app:srcCompat="@drawable/ic_eye_24dp"
-        app:tint="@color/white" />
+        app:tint="?android:attr/textColorSecondary" />
```

Comment:
Why textColorSecondary here?

I ask because `item_image_preview_overlay` (the "ALT" indicator) uses `@color/white`, so I would expect that this would be consistent with that.

Reply:
It doesn't, right? I remember changing it specifically to match ALT's:
https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/res/layout/item_image_preview_overlay.xml#L21

Reply:
Ah, I was looking at the `app:tint` for the alt indicator, not the text colour.